src/components/incidents/FilterForm.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Collapse, Form, Select, DatePicker } from 'antd';
+import { useSelector } from 'react-redux';
+
+const { Option } = Select;
+const { Panel } = Collapse;
+const { RangePicker } = DatePicker;
+
+const states = [
+  'Alabama',
+  'Alaska',
+  'Arizona',
+  'Arkansas',
+  'California',
+  'Colorado',
+  'Connecticut',
+  'Delaware',
+  'Florida',
+  'Georgia',
+  'Hawaii',
+  'Idaho',
+  'Illinois',
+  'Indiana',
+  'Iowa',
+  'Kansas',
+  'Kentucky',
+  'Louisiana',
+  'Maine',
+  'Maryland',
+  'Massachusetts',
+  'Michigan',
+  'Minnesota',
+  'Mississippi',
+  'Missouri',
+  'Montana',
+  'Nebraska',
+  'Nevada',
+  'New Hampshire',
+  'New Jersey',
+  'New Mexico',
+  'New York',
+  'North Carolina',
+  'North Dakota',
+  'Ohio',
+  'Oklahoma',
+  'Oregon',
+  'Pennsylvania',
+  'Rhode Island',
+  'South Carolina',
+  'South Dakota',
+  'Tennessee',
+  'Texas',
+  'Utah',
+  'Vermont',
+  'Virginia',
+  'Washington',
+  'West Virginia',
+  'Wisconsin',
+  'Wyoming',
+];
```

Comment:
What about Washington DC?

In src\components\graphs\assets\bargraphAssets.js there's already a list of states. You could export/import that and use ```states.map(s => s.name)``` to get the states.

Reply:
Sure I can do that.